evap/staff/templates/staff_user_merge.html

```diff
@@ -0,0 +1,267 @@
+                <td{% if participation_list|length > 0 %} class="info"{% endif %}>
+                    {% for courses in participation_list %}
+                        {{ courses.grouper }}
+                        <ul>
+                            {% for course in courses.list %}
+                                <li>{{ course.name }}</li>
+                            {% endfor %}
+                        </ul>
+                    {% endfor %}
+                </td>
+                {% regroup merged_user.participant_in by semester as participation_list %}
+                <td{% if participation_list|length > 0 %} class="success"{% endif %}>
+                    {% for courses in participation_list %}
+                        {{ courses.grouper }}
+                        <ul>
+                            {% for course in courses.list %}
+                                <li>{{ course.name }}</li>
+                            {% endfor %}
+                        </ul>
+                    {% endfor %}
+                </td>
+            </tr>
+            <tr{% if "voted_for" in errors %} class="danger"{% endif %}>
+                <td><strong>{% trans "Voted for" %}</strong></td>
+                {% regroup main_user.get_sorted_courses_voted_for by semester as voting_list %}
+                <td{% if voting_list|length > 0 %} class="info"{% endif %}>
+                    {% for courses in voting_list %}
+                        {{ courses.grouper }}
+                        <ul>
+                            {% for course in courses.list %}
+                                <li>{{ course.name }}</li>
+                            {% endfor %}
+                        </ul>
+                    {% endfor %}
+                </td>
+                {% regroup other_user.get_sorted_courses_voted_for by semester as voting_list %}
+                <td{% if voting_list|length > 0 %} class="info"{% endif %}>
+                    {% for courses in voting_list %}
+                        {{ courses.grouper }}
+                        <ul>
+                            {% for course in courses.list %}
+                                <li>{{ course.name }}</li>
+                            {% endfor %}
+                        </ul>
+                    {% endfor %}
+                </td>
+                {% regroup merged_user.voted_for by semester as voting_list %}
+                <td{% if voting_list|length > 0 %} class="success"{% endif %}>
+                    {% for courses in voting_list %}
+                        {{ courses.grouper }}
+                        <ul>
+                            {% for course in courses.list %}
+                                <li>{{ course.name }}</li>
+                            {% endfor %}
+                        </ul>
+                    {% endfor %}
+                </td>
+            </tr>
+        </tbody>
+    </table>
```

Comment:
man this really is a huge amount of copy-pasta. but i couldn't see a good way to reduce it, let's talk about this tomorrow some more